sdk/python/feast/repo_config.py

```diff
@@ -100,18 +109,20 @@ def _validate_online_store_config(cls, values):
                     values["online_store"]["type"] = "sqlite"
                 elif values["provider"] == "gcp":
                     values["online_store"]["type"] = "datastore"
-
+                elif values["provider"] == "aws_dynamo":
+                    values["online_store"]["type"] = "dynamo"
             online_store_type = values["online_store"]["type"]
-
             # Make sure the user hasn't provided the wrong type
-            assert online_store_type in ["datastore", "sqlite"]
+            assert online_store_type in ["datastore", "sqlite", "dynamo"]
```

Comment:
I believe we can also modify it to dynamodb to distinguish between dynamo and dynamodb.